cmd/ndm_daemonset/probe/uuid.go

```diff
@@ -117,3 +126,12 @@ func generateUUIDFromPartitionTable(bd blockdevice.BlockDevice) (string, bool) {
 	}
 	return "", false
 }
+
+// TODO move it to some other place
+func isDM(devPath string) bool {
+	devName := strings.Replace(devPath, "/dev/", "", 1)
```

Comment:
do we need to replace? we can check `if devName[0:8] == "/dev/dm-"`?

Reply:
Done.